pycontrails/models/cocipgrid/cocip_grid.py

```diff
@@ -441,6 +441,8 @@ def _set_timesteps(self) -> None:
 
         t_start = tmin.ceil(dt)
         t_end = tmax.floor(dt) + self.params["max_age"] + dt
+        _check_met_rad_time(self.met, self.rad, tmin, t_end)
```

Comment:
For consistency, use `tmin` and `tmax` or `t_start` and `t_end`? Maybe you have a reason for doing this, though.

Otherwise, re-incorporated warnings look good!

Reply:
Good eye! I wanted to use the minimum and the maximum of the four variables `tmin`, `tmax`, `t_start`, `t_end`, which is why you see the inconsistency here. I'll leave a comment here.

Reply:
9f81078e61cf9a1df9976d4bbf354d4521555e2c